etc/config/llvm_mir.amazon.properties

```diff
@@ -0,0 +1,65 @@
+compilers=&mirllc
+defaultCompiler=mirllctrunk
+
+demangler=/opt/compiler-explorer/gcc-11.1.0/bin/c++filt
+objdumper=/opt/compiler-explorer/gcc-11.1.0/bin/objdump
+
+licenseName=LLVM Apache 2
+licenseLink=https://github.com/llvm/llvm-project/blob/main/LICENSE.TXT
+licensePreamble=The LLVM Project is under the Apache License v2.0 with LLVM Exceptions
+
+group.mirllc.compilers=mirllc32:mirllc33:mirllc391:mirllc400:mirllc401:mirllc500:mirllc600:mirllc700:mirllc800:mirllc900:mirllc1000:mirllc1001:mirllc1100:mirllc1101:mirllc1200:mirllc1201:mirllc1300:mirllc1400:mirllc1500:mirllctrunk:mirllc-assertions-trunk
+group.mirllc.compilerType=llc
```

Comment:
If this uses `llc`, I think you'll need to set `.compilerType=llc` for the group as @matsjla figured out

Reply:
I didn't get you. What change should I make in ```group.mirllc.compilerType=llc``` ?

Reply:
Oh, nothing. Github app bugged and I didn't actually see that this line was as already should be! So all good to go :)